lib/v2/misskey/templates/note.art

```diff
@@ -0,0 +1,21 @@
+{{ text.replace(/\n/g, '<br>') }}
```

Comment:
Misskey's documentation says that [`text` is nullable](https://misskey-hub.net/docs/api/entity/note.html), so this line may throw an error.

Reply:
Reproduced. I will fix it now.